app/src/main/java/com/goayo/debtify/viewModel/GroupsViewModel.java

```diff
@@ -20,11 +21,16 @@
  * @date 2020-09-25
  */
 
-public class GroupsViewModel extends ViewModel {
+public class GroupsViewModel extends ViewModel implements IObserver {
```

Comment:
not sure if this is gonna be merge conflict but in #81 this was renamed to MyGroupsViewModel